script.js

```diff
@@ -0,0 +1,58 @@
+      $(document).ready(function(){
```

Comment:
because your script tag is just before the closing body, `$(document).ready` is not required